cmd/serve.go

```diff
@@ -43,7 +43,12 @@ on configuration options, open the configuration documentation:
 >> https://www.ory.sh/docs/oathkeeper/configuration <<
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		watchAndValidateViper()
+		logger = viperx.InitializeConfig("oathkeeper", "", logger)
+		schema, err := schemas.Find("config.schema.json")
+		if err != nil {
+			logger.WithError(err).Fatal("Unable to open configuration JSON Schema.")
+		}
+		viperx.WatchAndValidateViper(logger, schema, "ORY Oathkeeper", []string{"serve", "profiling", "log"})
```

Comment:
Move this back to its own function (preferably `watchAndValidateViper`) so that it can be reused in other tasks as well if neccessary